provider/cmd/crd2pulumi/README.md

```diff
@@ -1,148 +1,243 @@
 
-		constructor(name: string, args?: v1.CronTabArgs, opts?: pulumi.CustomResourceOptions) {
-			super(name, { apiVersion: "stable.example.com/v1", kind: "CronTab", ...args }, opts)
-		}
-	}
-}
+Now let's run the program and perform the update.
 
-export class CronTabDefinition extends k8s.apiextensions.v1.CustomResourceDefinition {
-	constructor(name: string, opts?: pulumi.CustomResourceOptions) {
-		super(name, {
-			"apiVersion": "apiextensions.k8s.io/v1",
-			"kind": "CustomResourceDefinition",
-			"metadata": {
-				"name": "crontabs.stable.example.com"
-			},
-			"spec": {
-				"group": "stable.example.com",
-				"names": {
-					"kind": "CronTab",
-					"plural": "crontabs",
-					"shortNames": [
-						"ct"
-					],
-					"singular": "crontab"
-				},
-				"scope": "Namespaced",
-				"versions": [
-					{
-						"name": "v1",
-						"schema": {
-							"openAPIV3Schema": {
-								"properties": {
-									"spec": {
-										"properties": {
-											"cronSpec": {
-												"type": "string"
-											},
-											"image": {
-												"type": "string"
-											},
-											"replicas": {
-												"type": "integer"
-											}
-										},
-										"type": "object"
-									}
-								},
-								"type": "object"
-							}
-						},
-						"served": true,
-						"storage": true
-					}
-				]
-			}
-		}, opts)
-	}
-}
+```bash
+$ pulumi up
 ```
+
+```bash
+Previewing update (dev):
+  Type                                                      Name                Plan
+  pulumi:pulumi:Stack                                       examples-dev
+ +   ├─ kubernetes:stable.example.com:CronTab                   my-new-cron-object  create
+ +   └─ kubernetes:apiextensions.k8s.io:CustomResourceDefinition  my-crontab-definition  create
+Resources:
+  + 2 to create
+  1 unchanged
+Do you want to perform this update?  [Use arrows to move, enter to select, type to filter]
+> yes
+  no
+  details
+```
+
+```bash
+Do you want to perform this update? yes
+Updating (dev):
+  Type                                                      Name                Status
+  pulumi:pulumi:Stack                                       examples-dev
+ +   ├─ kubernetes:stable.example.com:CronTab                   my-new-cron-object  created
+ +   └─ kubernetes:apiextensions.k8s.io:CustomResourceDefinition  my-crontab-definition  created
+Outputs:
+  urn: "urn:pulumi:dev::examples::kubernetes:stable.example.com/v1:CronTab::my-new-cron-object"
+Resources:
+  + 2 created
+  1 unchanged
+Duration: 17s
+Permalink: https://app.pulumi.com/albert-zhong/examples/dev/updates/4
+```
+
+It looks like both the CronTab definition and instance were both created! Finally, let's verify that they were created
+by manually viewing the raw YAML data:
+
+```bash
+$ kubectl get ct -o yaml
+```
+
+```bash
```

Comment:
I realize this is the output from the shell, so "bash" is technically correct, but since you have it as a separate code block, wondering if it'd render nicer as "yaml".

```suggestion
```yaml
```